docs/endpoints/post-token-refresh.md

```diff
@@ -19,9 +19,8 @@ Used by: This endpoint is used mainly by publishers.
 Here's what you need to know about this endpoint:
 
 - No encryption is required for token refresh requests.
-- Responses are encrypted only if the HTTP status code is 200. Otherwise, responses are not encrypted.
-- To decrypt responses, the `refresh_response_key` value returned in the [POST /token/generate](post-token-generate.md) or `POST /token/refresh` response from which the refresh token in the request is returned.
-- If you send a refresh token from a v1 `token/generate` response in the request, the response will not be encrypted.
+- If the request is successful, with an HTTP status code of 200 and a new UID2 token or opt-out information returned, the response is encrypted. Error responses are not encrypted. [**GWH/SW_01 opt-out is a 200 response but I'm not sure whether it's encrypted or not. It's mentioned 3 times in this doc. Could we clarify please whether an opt-out response is encrypted?**]
```

Comment:
Optout response is encrypted according to [here ](https://github.com/IABTechLab/uid2-operator/blob/master/src/main/java/com/uid2/operator/vertx/UIDOperatorVerticle.java#L393) and [here](https://github.com/IABTechLab/uid2-operator/blob/master/src/main/java/com/uid2/operator/vertx/V2PayloadHandler.java#L132)

Reply:
Same with the Java SDK - it decrypts any 2xx response